### PR TITLE
Add a bio for elezar

### PR DESCRIPTION
**Pull request by elezar**
Add a short bio for @elezar.

How does it look @frontendherodk?

### CODE REVIEW

File: profiles/elezar.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Evan Lezar
+![](https://github.com/elezar.png)
+
+## About me
+A generalist that likes pina coladas and dancing in the rain. Except the rain part. I hate that.
```
Comment (frontendherodk) [inline profiles/elezar.md:5]: What if it is purple rain?

:shipit:

**Comment by elezar** [comment]
@brntbeer @mwiesen @stoe how about a merge? @frontendherodk says that it looks good!

(Or should I just merge it myself?)

**Comment by mwiesen** [comment]
@elezar looks good to me! :shipit: !

File: profiles/elezar.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+## TODOs for today:
+- [x] Have coffee
+- [ ] Attend GitHub for Developers training
+- [ ] Have more coffe
```
Comment (hesserp) [inline profiles/elezar.md]: @elezar typo: it should be coffee

**Comment by frontendherodk** [comment]
@elezar You should merge!

File: profiles/elezar.md (hunk 2 above)
Comment (elezar) [inline profiles/elezar.md]: Thanks @hesserp. Let me fix that

**Comment by elezar** [comment]
@frontendherodk @hesserp Just look over one more time. I fixed the spelling mistake.

**Comment by frontendherodk** [comment]
🚢

**Pull request merged**